drivers/input/Kconfig

```diff
@@ -0,0 +1,36 @@
+# Input subsystem configuration options
+
+# Copyright (c) 2022, tangchunhui@coros.com
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig INPUT
+	bool "Input Subsystem Drivers"
+	select RING_BUFFER
+	help
+	  Include input subsystem drivers in system config.
+
+if INPUT
+
+config ENABLE_INPUT_ISR_LOCK
+	bool "Input event interrupt lock"
+	help
+	  Use isr_lock() and isr_unlock() to control ring buffer exclusive access.
```

Comment:
I think this should be converted to a spinlock. See [Legacy irq_lock() emulation](https://docs.zephyrproject.org/3.2.0/kernel/services/smp/smp.html#legacy-irq-lock-emulation) for more information.